packages/flutter/lib/src/material/tabs.dart

```diff
@@ -849,6 +855,13 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
   /// ```
   final InteractiveInkFeatureFactory? splashFactory;
 
+  /// Defines the clipping radius of splashes that extend outside the bounds of the tab.
+  ///
+  /// For example, to match the [BoxDecoration.borderRadius] provided as [indicator].
```

Comment:
```suggestion
  /// This can be useful if you want to match the [BoxDecoration.borderRadius] provided as [indicator].
```

Reply:
Should I also change 'provided as' to 'provided in' ?

Reply:
And as per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require should I do 'This can be useful' ?